tests/testthat/_snaps/run_stress_test.md

```diff
@@ -0,0 +1,6 @@
+# with multiple inputs to non-iterable arguments errors gracefully
+
+    All input arguments need to hold values.
+    x Missing arguments: input_path_project_specific, input_path_project_agnostic, output_path.
+    i Did you provide all input arguemnts correctly?.
```

Comment:
FIXME: This error message is not what we want if we run `run_stress_test(asset_type = 1:2)`.

This highlights the need for automated tests ASAP.